apps/web/app/reader/route.ts

```
@@ -0,0 +1,196 @@
+      throw new Error(`Jina API responded with status: ${response.status}`);
+    }
+    
+    // First try to get the response as text to check for binary content
+    const responseText = await response.text();
+    
+    // Check if the response is actually JSON
+    try {
+      const data = JSON.parse(responseText);
+      
+      // Check if content exists and sanitize it
+      if (data.data && data.data.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)